app/src/test/java/org/akvo/flow/ui/model/ViewFormMapperTest.java

```diff
@@ -122,7 +122,6 @@ public void transformShouldTransformCorrectlyAllSurveyFields() {
         given(mockSurveyItem.isRegistrationForm()).willReturn(true);
         given(mockSurveyItem.getId()).willReturn("123");
         given(mockSurveyItem.getName()).willReturn("name");
-        given(mockSurveyItem.isSubmittedDataPoint()).willReturn(true);
```

Comment:
Why we don't expect the datapoint now?

Reply:
No, that was marked as "not needed mock by JUnit"